app/services/course/assessment/question/programming/cpp/cpp_package_service.rb

```diff
@@ -0,0 +1,219 @@
+# frozen_string_literal: true
+class Course::Assessment::Question::Programming::Cpp::CppPackageService < \
+  Course::Assessment::Question::Programming::LanguagePackageService
+  AUTOGRADE_INCLUDE_PATH = File.join(File.expand_path(File.dirname(__FILE__)),
```

Comment:
If this is repeated a lot, can also refactor this and place it somewhere to be included.